scripts/removeBills.js

```diff
@@ -0,0 +1,42 @@
+const mkAPI = require('./api')
+
+const DOCTYPE_BILLS = 'io.cozy.bills'
+const get = require('lodash/get')
+
+// define your bills filter here
+// I am sure there is a better wait to configure that in ACH
+//
+function billsToRemoveFilter(bill) {
+  // return bill.vendor === 'impot'
+  // return get(bills.cozyMetadata.createdByApp) === 'impot' && get(bills.cozyMetadata.createdByAppVersion === "1.9.0")
```

Comment:
Would be interesting to do that directly with a mango selector.

```
ACH script removeDocuments '{ cozyMetadata: { createdByApp: 'impot', createdByAppVersion: '1.9.0' } }'
```

[sift](https://www.npmjs.com/package/sift) can be useful for this.